### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by jaswanth115**
Html home task

### CODE REVIEW

**Comment by jaswanth115** [comment]
Please review.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    </div>
+    
+    <div class="row1">
+        <div class="r1col1">
+            <form action="/action_page.php">
+                <label>SELECT CATEGORY:</label>
+                <select id="select" onchange="tagchange()">
+                    <option value="All" href="#general" id="general">All</option>
+                    <option value="Sports" href="#sports" id="sport">Sports</option>
+                    <option value="Business" href="#business" id="business">Business</option>
```
Comment (NirmalaEpam) [inline index.html:24]: create a constant file and get the string value from that file

File: index.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        </div>
+        <div class="r1col2">
+            <h2 id="tag">All News</h2>
+        </div>
+    </div>
+
+    <!-- for news -->
+    <div class="news_row" >
+        <div class="fig news_row">
+            <img src="https://i.postimg.cc/kMV1Z9xR/Screenshot-37.png" alt="Chanell Logo" class="fig">
```
Comment (NirmalaEpam) [inline index.html:38]: use the images from local folder by downloading it .

File: index.html [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+            <h3 class="heading">Title One</h3>
+            <p class="time" id="date1"></p>
+            <p class="para" id="para1"></p>
+            <button class="but">Continue Reading</button>
+       
+        </div>
+    </div>
+    <div class="news_row2" >
+        <div class="fig news_row">
+            <img src="https://i.postimg.cc/kMV1Z9xR/Screenshot-37.png" alt="Chanell Logo" class="fig">
```
Comment (NirmalaEpam) [inline index.html:50]: follow the same for images as above

File: index.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    document.getElementById("tag").innerHTML = x+b;  
+}
+function date(){
+    const d = new Date();
+    document.getElementById("date1").innerHTML = d;
+    document.getElementById("date2").innerHTML = d;
+    document.getElementById("date3").innerHTML = d;
+}
+function paragraph(){
+    const d = "Delhi is a crowded city. There are very few rich people who travel by their own vehicles.";
```
Comment (NirmalaEpam) [inline index.js:14]: change variable name and pass the text to constant file

**Pull request merged**